packages/runner/lib/abort.ts

```diff
@@ -5,6 +5,7 @@ export const abort = (taskId: string): boolean => {
     const ac = abortControllers.get(taskId);
     if (ac) {
         ac.abort();
+        abortControllers.delete(taskId);
```

Comment:
unrelated to this issue but I realized we were not very good citizen, not removing the abortController from the global map once it has been aborted